flow/activities/flowable.go

```diff
@@ -283,16 +283,24 @@ func (a *FlowableActivity) StartFlow(ctx context.Context,
 			return nil, fmt.Errorf("failed in pull records when: %w", err)
 		}
 		slog.InfoContext(ctx, "no records to push")
-		syncResponse := &model.SyncResponse{}
-		syncResponse.RelationMessageMapping = <-recordBatch.RelationMessageMapping
-		syncResponse.TableSchemaDeltas = recordBatch.WaitForSchemaDeltas(input.FlowConnectionConfigs.TableMappings)
-		return syncResponse, nil
+		tableSchemaDeltas := recordBatch.WaitForSchemaDeltas(input.FlowConnectionConfigs.TableMappings)
+		if normConn, ok := dstConn.(connectors.CDCNormalizeConnector); ok {
```

Comment:
can we move replay schema deltas to CDCSyncConnector?

Reply:
Not without creating nop methods on CDCSyncConnectors for s3/eventhub

Reply:
Discussed oob; preference is for nop methods